AdvancedGuide.md

```diff
@@ -30,6 +30,37 @@ Yes it can. Form title has to be in English, but instructions and fields can be
 
 The current size limit is 7 MB for the entire form. We auto-compress images to 1024x768 resolution, which is typically less than 1 MB. This is a hard limit because the email service we use has a fixed 10 MB outgoing size, and we buffer 3 MB for email fields and metadata. Because the smallest unit you can attach per attachment field is 1 MB, you can have a max of 7 attachments on your form. If your user has to submit more than 7 documents, you may create just one attachment field of 7 MB, and advise your user to zip documents up and submit as one attachment. Now that Storage mode is launched, we have plans in Q3 2020 to increase attachment size limit to 20 MB.
 
+## Prefill
+
+### What is prefill?
+
+Prefill is a convenience feature that allows you to pre-populate fields for your form user. This adds convenience and reduces the chance of mistakes in manual entry. Examples where prefill has been used include populating form serial numbers.
```

Comment:
can change this section to say:

Selecting 'Enable pre-fill' on a form field allows the field value to be pre-filled or pre-populated when a public form is loaded, before the respondent has entered any value. This is done by adding the pre-filled value in a specific format to the public form's URL which the respondent accesses.

e.g. For a pre-filled value "Yew Tee Primary School"
https://form.gov.sg/6079216d5efcf5002c523299/?6080d2ab13daf3002bfcd32d=Yew+Tee+Primary+School

This feature adds convenience for the respondent when field values are known beforehand. For example, a customer service ID can be pre-filled and added to a URL before sending it to a customer for feedback; or the form URL made available to respondents from a specific school or location can be customised to pre-fill that school or location in the relevant field.

Pre-fill is currently only available on Short Text fields and may be released for other fields in future, depending on demand."